Passwords/twitter-banned.txt

```diff
@@ -126,7 +126,6 @@ diamond
 doctor
 doggie
 dolphin
```

Comment:
remove this typo

Reply:
god, I missed this 😂

Reply:
I think it's completely different password which has no connection with PR topic.

Reply:
Yes @luongvo - you did.

Reply:
:shipit:

Reply:
hi @god, have a nice day 🚀

Reply:
PLEASE REMOVE THIS AS WELL, as it is a sECuRItY hole.

Reply:
Please let this man live knowing he can no longer be hacked.